### PR TITLE
BaSP-MR-160: [Member] Create class schedule

### PR DESCRIPTION
**Pull request by EmaBaldor**
Implement the necessary logic and design elements to create a visually appealing screen that showcases the data retrieved from the database for "Classes". The data should be presented in a calendar-style table format.

### CODE REVIEW

**Comment by EmaBaldor** [comment]
[[Member] Create class schedule component](https://trello.com/c/HnW8YlSW/160-member-create-class-schedule-component)

File: src/Components/Member/Schedule/index.jsx [hunk 1]
```diff
@@ -1,7 +1,150 @@
-import React from 'react';
+import { useEffect, useState } from 'react';
+import styles from './schedule.module.css';
+import { useDispatch, useSelector } from 'react-redux';
+import { BiStar } from 'react-icons/bi';
+import { getClasses } from '../../../Redux/classes/thunks';
+import { getActivities } from '../../../Redux/activities/thunks';
+import { getMembersById } from '../../../Redux/members/thunks';
+import { getSubscriptions } from '../../../Redux/subscriptions/thunks';
```
Comment (Federico-Peralta) [inline src/Components/Member/Schedule/index.jsx]: You should change the imports relative to absolute

**Review by Federico-Peralta — DISMISSED** [review]

File: src/Components/Member/Schedule/index.jsx [hunk 2]
```diff
@@ -1,7 +1,147 @@
...
+
+  const [memberData, setMemberData] = useState(null);
+  const [suscriptionsMember, setSuscriptionsMember] = useState('');
+  const [activity, setActivity] = useState('');
+
+  const daysOfWeek = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+
+  const hoursOfDay = [
+    { label: '8:00', value: 8 },
+    { label: '9:00', value: 9 },
```
Comment (Federico-Peralta) [inline src/Components/Member/Schedule/index.jsx]: If you don't add a 0 on the label in won't show the classes that are on 08:00 and 09:00

**Review by Gonzarey96 — APPROVED** [review]
The code looks good and works, congratulations.

**Review by ale-schmidt — APPROVED** [review]
Looks and works great, congratulations!

**Review by AleMac17 — APPROVED** [review]

**Review by fernandofernandezuy — APPROVED** [review]
Amazing job Ema!

**Comment by araefedg** [comment]
Everything implemented seems to work correctly! Great work!

**Pull request merged**